src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestTrustedMasterKeyPaths.cs

```diff
@@ -152,8 +152,7 @@ public void TestTrustedColumnEncryptionMasterKeyPathsWithMultipleServers(string
             // Add some random key paths
             foreach (char c in new char[] { 'A', 'B' })
             {
-                string tempThumbprint = new string('F', CertificateUtility.CreateCertificate().Thumbprint.Length);
-                string invalidKeyPath = string.Format(@"{0}/my/{1}", StoreLocation.CurrentUser.ToString(), tempThumbprint);
+                string invalidKeyPath = string.Format(@"{0}/my/{1}", StoreLocation.CurrentUser.ToString(), dummyThumbprint);
```

Comment:
I don't see where `c` is being used int this (or the original) - can we either have the values be useful or just make it be a for loop? (or remove it if it's not useful)

Reply:
It's not - slightly odd code. I've replaced it with a collection initializer.